extensions/mssql/package.json

```diff
@@ -919,15 +919,66 @@
           "categoryValues": [
             {
               "displayName": "%mssql.disabled%",
-              "name": "Disabled"
+              "name": "disabled"
             },
             {
               "displayName": "%mssql.enabled%",
-              "name": "Enabled"
+              "name": "enabled"
             }
           ],
           "isRequired": false,
-          "isArray": false
+          "isArray": false,
+          "onSelectionChange": [
+            {
+              "values": [
+                "disabled",
+                ""
+              ],
+              "dependentOptionActions": [
+                {
+                  "optionName": "secureEnclaves",
+                  "action": "hide"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "secureEnclaves",
+          "displayName": "%mssql.connectionOptions.secureEnclaves.displayName%",
+          "description": "%mssql.connectionOptions.secureEnclaves.description%",
+          "groupName": "%mssql.connectionOptions.groupName.security%",
+          "valueType": "category",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": [
+            {
+              "displayName": "%mssql.disabled%",
+              "name": "disabled"
+            },
+            {
+              "displayName": "%mssql.enabled%",
+              "name": "enabled"
+            }
+          ],
+          "isRequired": false,
+          "isArray": false,
+          "onSelectionChange": [
+            {
+              "values": [
+                "disabled",
+                ""
+              ],
+              "dependentOptionActions": [
+                {
+                  "optionName": "attestationProtocol",
+                  "action": "hide"
```

Comment:
when the fields are being hid, do we clear the data in the those fields?

Reply:
Yes, they're reset to default (empty) when hidden